app/cinema/hall.py

```diff
@@ -0,0 +1,21 @@
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> int:
+        self.number = number
+
+    def movie_session(self,
+                      movie_name: str,
+                      customers: [Customer],
```

Comment:
```suggestion
                      customers: list[Customer],
```
[Customer] does not mean list of Customer
Or you can use List[Customer], where List is from typing library